guides/release/components/index.md

```diff
@@ -142,4 +142,5 @@ In the following guides we'll talk about:
   which Ember fully supports
```

Comment:
While at id, this should have a period at the end to follow the style of the rest of this list.